server/core/src/main/java/io/eiren/math/FloatMath.kt

```diff
@@ -17,7 +17,7 @@ object FloatMath {
 	const val ZERO_TOLERANCE_D: Double = 0.0001
 
 	val SQRT_TWO: Float = kotlin.math.sqrt(2.0).toFloat()
-	val INV_SQRT_TWO: Float = 1f / SQRT_TWO
+	val INV_SQRT_TWO: Float = (1.0 / kotlin.math.sqrt(2.0)).toFloat()
```

Comment:
Why change this?

Reply:
`1f / SQRT_TWO` guarantees the error is less or equal to 1.5 ULP.

`(1.0 / kotlin.math.sqrt(2.0)).toFloat()` guarantees the error is less or equal to 0.5 ULP.

Reply:
Cool I thought it would be something like that